### PR TITLE
added pontoon.js when L10N_DEV=true

### PR DESCRIPTION
**Pull request by dannycoates**
Let's see if this works!

cc @flodolo

### CODE REVIEW

**Comment by pdehaan** [comment]
Unrelated to specific issue, but something funny happening w/ package-lock.json diffs in these recent PRs.
Not sure if we're all on different versions of npm@5, but we're seeing a +/-2000 line change on every odd commit.

File: server/server.js [hunk 1]
```diff
@@ -49,39 +49,58 @@ app.engine(
...
+    frameAncestors: ["'none'"],
+    objectSrc: ["'none'"],
+    reportUri: '/__cspreport__'
+  }
+};
+
+if (conf.l10n_dev) {
+  const d = cspPolicy.directives;
+  d.scriptSrc.push('https://pontoon.mozilla.org');
+  d.imgSrc.push('https://pontoon.mozilla.org');
```
Comment (pdehaan) [inline server/server.js]: Do we load images/styles from pontoon?

**Comment by flodolo** [comment]
You want to talk with @mathjazz for this, I'm not familiar with the process to set up in-content localization.

**Comment by mathjazz** [comment]
Sadly, I'm not exactly sure this will add a lot of value to the l10n process. Pontoon doesn't offer anything ATM to navigate between different page views, except for links to subpages. Since this is a SPA, the users would only have an easy access to the homepage.

If you still want to give it a shot, I suggest we use the stage server. In which case all the URLs in the [docs (Section B)](https://developer.mozilla.org/en-US/docs/Mozilla/Implementing_Pontoon_in_a_Mozilla_website) need to be replaced with the stage URL - mozilla-pontoon-staging.herokuapp.com (which includes the path to the `pontoon.js` file in this patch).

**Comment by dannycoates** [comment]
ok, let's table this for now

**Pull request closed without merging**